documentation/modules/exploit/linux/http/panos_auth_rce.md

```diff
@@ -0,0 +1,47 @@
+This module exploits an authenticated OS Command Injection vulnerability in PAN-OS versions: <10.0.1, <9.1.4 and <9.0.10
+
+## Vulnerable Application
+
+A Palo Alto Firewall demo can be requested at the following [link](https://www.paloaltonetworks.com/company/request-demo)
+
+## Verification Steps
+
+1. Install the application
```

Comment:
Given the complexity of this target and potential for the need to have specific configurations for various exploits, I'd like to see some more information on setting up a target here vs just a "draw the rest of the owl" type of one liner like this.

Any chance we could get even a brief tutorial? I'm happy to assist on writing this up, just don't think its appropriate to not include at least some information here.

Reply:
This was copied from else where and I do agree it is too vague. Although - the virtual machine that you get from Palo Alto is a Red Hat VM with PAN-OS already running on top of it. There isn't much configuration necessary, apart for possibly some networking configuration although VMware Fusion handled that for me, running on something else like VirtualBox might be different.

I was able to browse to the web interface as soon as I powered up the VM for the first time. I also don't like it when installation instructions are glossed over although I don't _think_ much more needs to be said here. I would be happy to add more details if you think of any when you spin this up. Thanks for pointing this out!